strr-examiner-web/app/stores/examiner.ts

```diff
@@ -162,6 +164,38 @@ export const useExaminerStore = defineStore('strr/examiner-store', () => {
     })
   }
 
+  /**
+   * Assign the current user as the reviewer of an application.
+   *
+   * @param {string} applicationNumber - The application number to assign.
+   */
+  const assignApplication = async (applicationNumber: string): Promise<void> => {
```

Comment:
Just curious how does it work if no IDIR is being passed?

Reply:
401 unauthorized.
Only these roles are allowed:
`@jwt.has_one_of_roles([Role.STRR_EXAMINER.value, Role.STRR_INVESTIGATOR.value])`

Reply:
I meant how it knows which IDIR to assign to application if we do not pass it in payload?

Reply:
Uses the current user IDIR. 
If application is unassigned, an examiner opens the application then it gets assigned to that application.
https://github.com/bcgov/STRR/blob/main/strr-base-web/app/plugins/strr-api.ts